static/styles/input_pill.css

```diff
@@ -112,6 +112,14 @@
     }
 }
 
+.search_list_item .pill-container {
+    margin-left: 5px;
+
+    .exit {
```

Comment:
Let's add a comment for this, noting that the purpose is to disable the X to remove pill. 

Or, a better fix might be to extend the pill library to have an option to just not include an exit element.